maap/maap.py

```diff
@@ -256,13 +257,12 @@ def cancelJob(self, jobid):
         job.id = jobid
         return job.cancel_job()
 
-    def listJobs(self, username=None, 
+    def listJobs(self, username=None, *,
```

Comment:
What is this `*` for ?

Reply:
The `*` means that any parameters that appear _after_ it in the parameter list _must_ be supplied using keywords. In this case, `username` is the _only_ parameter that can be supplied _without_ a keyword, but _everything else_ must be supplied with keywords. This is _very_ helpful when there are many possible options, but you don't want to have to pass _every_ argument in the correct order, supplying `None` value for args you don't care about.